roles/logging-config/handlers/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 - name: restart logstash-forwarder
   service: name=logstash-forwarder state=restarted
+  when: ansible_architecture != "ppc64le"
```

Comment:
Then lets guard these based on a fact logstash is available on the current architecture, something which may change.

If we want to target PPC we should port required packages to PPC rather than cut out everything which doesn't fit.